FlowCrypt/Controllers/Threads/ThreadDetailsViewController+TapActions.swift

```diff
@@ -119,7 +125,39 @@ extension ThreadDetailsViewController {
 
 // MARK: - Compsoe
 extension ThreadDetailsViewController {
-    private func composeNewMessage(at indexPath: IndexPath, quoteType: MessageQuoteType) {
+    private func fetchAttachmentDataIfNil(attachment: MessageAttachment, messageId: Identifier) async throws -> MessageAttachment {
+        if attachment.data == nil {
+            var updatedAttachment = attachment
+            showSpinner()
+            updatedAttachment.data = try await messageHelper.download(
+                attachment: attachment,
+                messageId: messageId,
+                progressHandler: { [weak self] progress in
+                    self?.handleFetchProgress(state: .download(progress))
+                }
+            )
+            return updatedAttachment
+        }
+        return attachment
+    }
+
+    private func decryptAttachment(for attachment: MessageAttachment) async throws -> MessageAttachment {
```

Comment:
It can be just `decrypt(attachment: MessageAttachment)`, as `decryptAttachment(for: attachment)` looks strange

Reply:
ok